src/localconf.js

```diff
@@ -104,6 +104,33 @@ function buildConfig(opts) {
     useGuppyForExplorer = true;
   }
 
+  let explorerConfig = [];
+  // for backward compatibilities
+  if (config.dataExplorerConfig) {
+    explorerConfig.push(
+      {
+        tabTitle: 'Data',
+        ...config.dataExplorerConfig,
+      },
+    );
+  }
+  if (config.fileExplorerConfig) {
+    explorerConfig.push(
+      {
+        tabTitle: 'File',
+        ...config.fileExplorerConfig,
+      },
+    );
+  }
+
+  // new explorer config format
+  if (config.explorerConfig) {
+    useGuppyForExplorer = true;
+    explorerConfig = config.explorerConfig;
```

Comment:
oh i think i didn't explain that clearly. The logic should be if there is the new config `explorerConfig` than we just ignore old `dataExplorerConfig` and `fileExplorerConfig` if any. But if thre is no `explorerConfig` then the code is backward compatiable with `dataExplorerConfig` and `fileExplorerConfig`

I wasn't mean to support the both new and old configs in the same config file. Personally I feel that is redundent: if you want to use the new feature, migrate the config, otherwise you can just sit and relax. 😃 

I'll put that into the doc

Reply:
yeah i agree!